neon-animated-pages.html

```diff
@@ -86,19 +86,17 @@
 
     },
 
-    observers: [
-      '_selectedChanged(selected)'
-    ],
-
     listeners: {
+      'iron-select': '_onIronSelect',
       'neon-animation-finish': '_onNeonAnimationFinish'
     },
 
-    _selectedChanged: function(selected) {
-
-      var selectedPage = this.selectedItem;
+    _onIronSelect: function(event) {
```

Comment:
BTW, the bug on the previous code was that the observer was referring to two properties (`this.selected`, `this.selectedItem`), but the parameter list only had 'selected. Therefore when the observer was executing, `selected` would have a value but `selectedItem` was still undefined.

Just changing as follows to could also fix various weirdness I was observing.
```
_selectChanged: function(selected, selectedItem) { ...}
```
Would that be a better fix than listening for iron-select?